pkgs/tools/typesetting/scdoc/default.nix

```diff
@@ -20,6 +20,10 @@ stdenv.mkDerivation (finalAttrs: {
   postPatch = ''
     substituteInPlace Makefile \
       --replace "-static" ""
+    substituteInPlace include/str.h \
+      --replace "struct str *str_create();" "struct str *str_create(void);"
+    substituteInPlace src/string.c \
+      --replace "struct str *str_create() {" "struct str *str_create(void) {"
```

Comment:
This is fixed upstream in https://git.sr.ht/~sircmpwn/scdoc/commit/1923fb61cd71182f05d029f280d056b21f66e8a0.